taxi_service/urls.py

```diff
@@ -21,5 +21,6 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("accounts/", include("django.contrib.auth.urls")),
     path("", include("taxi.urls", namespace="taxi")),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
The `static` function is used to serve static files during development. Remember that in production, static files should be served by a web server like Nginx or Apache, not Django.